src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -653,12 +644,9 @@ class BaseOptionsSelector extends Component {
                         </>
                     )}
                 </View>
-                {this.props.shouldShowReferralCTA && this.state.shouldShowReferralModal && (
+                {this.props.shouldShowReferralCTA && (
```

Comment:
Remove `shouldShowReferralCTA` prop. Instead use `!dismissedReferralBanners(this.props.referralContentType)`

Reply:
Actually ignore this comment. I think it's better to keep BaseOptionsSelector onyx-free